src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -59,17 +64,43 @@ private static void setupCommandTests(Application application) {
 
 		CommandsCounterVisitor commandsCounterVisitor = new CommandsCounterVisitor();
 		publisher.addSubscriber(new SubscribeCommandsCounterVisitor(commandsCounterVisitor, manager));
-		
+
 		application.addTest("Load secret command", new SelectLoadSecretCommandOptionListener());
 		application.addTest("Load rectangle command", new SelectRectangleCommandOptionListener());
 
 		application.addTest("Run command", new SelectRunCurrentCommandOptionListener(DriverFeature.getDriverManager()));
+		// Selecting another driver resets previous transformer commands for this driver.
+		// Translate
+		List<TransformerCommand> translateCommands = new ArrayList<>();
+		translateCommands.add(new TranslateCommand(50, 50));
+		ComplexTransformerCommand translateComplexCommand = new ComplexTransformerCommand(translateCommands);
+		application.addTest("Translate",
+				new SelectTransformCommandOptionListener(
+						DriverFeature.getDriverManager(), translateComplexCommand, "Translate"));
+
+		// Scale
+		List<TransformerCommand> scaleCommands = new ArrayList<>();
+		scaleCommands.add(new ScaleCommand(1.1, 0.9));
+		ComplexTransformerCommand scaleComplexCommand = new ComplexTransformerCommand(scaleCommands);
+		application.addTest("Scale",
+				new SelectTransformCommandOptionListener(
+						DriverFeature.getDriverManager(), scaleComplexCommand, "Scale"));
+
+		// Rotate
+		List<TransformerCommand> rotateCommands = new ArrayList<>();
+		rotateCommands.add(new RotateCommand(10));
+		ComplexTransformerCommand rotateComplexCommand = new ComplexTransformerCommand(rotateCommands);
```

Comment:
The TransformerDriver uses the decorator approach. Is ComplexTransformerCommand in this case actually necessary?